### ISSUE TITLE
trainset and testset

### ISSUE THREAD

**Issue by liuzitongtt** [una-dinosauria/human-motion-prediction#63]
1.Does the model have a training set and a test set, how is it divided, 
2.and the length of the input sequence during the final verification and the length of the input sequence during the training must be the same?
Hope you can answer my question, thank you very much

**Comment by Droliven** [comment]
among human 3.6m 7 subjects, the subject 5 is used to test, the others are used to train
